Feat/email template

colganmeanor opened the pull request:
Added: 

- html and CSS files for email template for creating new event

Commit:
Add html and css for new event email

Commit:
Add css file  for new event email

Commit:
Fix import of css styles into html doc

matthewjholmes reviewed (approved):
Looks great to me.

matthewjholmes commented on app/views/event_mailer/new_event_email.html.erb:
```diff
@@ -1,10 +1,27 @@
+      <h1>Do Goodr</h1>
+    </header>
+    <div class='email-message'>
+      <p> Congratulations <%= @event.organization_name %> - your volunteering opportunity has successfully been posted to Do Goodr! The details of this event are below:</p>
```
Will need to be `@event.organization.name`

matthewjholmes commented on app/views/event_mailer/new_event_email.html.erb:
```diff
@@ -1,10 +1,27 @@
+      <p> Address: <%= @event.address %> </p>
+      <p> Type of Event: <%= @event.category %> </p>
+      <p> Date: <%= @event.date %> </p>
+      <p> Time: <%= @event.start_time %> - <%= @event.end_time %>  </p>
```
We may want to format this a bit more extensively to not get an unweildy DateTime.

matthewjholmes commented on app/views/event_mailer/new_event_email.html.erb:
```diff
@@ -1,10 +1,27 @@
+      <p> Event Name: <%= @event.name %> </p>
+      <p> Address: <%= @event.address %> </p>
+      <p> Type of Event: <%= @event.category %> </p>
+      <p> Date: <%= @event.date %> </p>
```
No such attritribute--it's part of start_time

Commit:
Fix event organization name on line 13

Commit:
Attempt to fix date time formatting using strftime method

Commit:
Add inline styles to new event email, delete unused CSS file

Commit:
Fix format of date in email, added slash between day and year

Commit:
Change email template file back to erb

Commit:
Add font styling to email

colganmeanor committed:
Merge branch 'main' into feat/email-template

colganmeanor commented:
I think everything should be ready to merge now @matthewjholmes - please let me know if there is anything else that needs attention. 

 - Added inline styles / deleted assets directory with css file
 - fixed slash with event date formatting
 - added font family import to reflect same font as the FE app
